css/src/help-center.scss

```diff
@@ -0,0 +1,108 @@
+.wpseo-tab-video-container {
```

Comment:
When you only use a scss file as a partial a best practice is to prefix the file with an underscore. So it becomes `css/src/_help-center.scss`.